.env

```
@@ -0,0 +1,3 @@
+DB_URL = jdbc:mysql://localhost:3306/elephantfinancelab_be
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)